http-core/src/main/resources/reference.conf

```diff
@@ -307,6 +307,12 @@ pekko.http {
       # Fail the connection if a sent ping is not acknowledged within this timeout.
       # When zero the ping-interval is used, if set the value must be evenly divisible by less than or equal to the ping-interval.
       ping-timeout = 0s
+
+      # Configure the throttle for Reset Frames (https://github.com/apache/incubator-pekko-http/issues/332)
+      resets-throttle-cost = 100
+      resets-throttle-burst = 100
+      # setting resets-throttle-interval to 0s will disable the throttle
+      resets-throttle-interval = 1s
```

Comment:
Did we make an official decision about whether this mitigation should be enabled by default or not? My memory may be fuzzy but I thought we decided to not have it enabled by default but have the option so people can enable it (in the case they expose pekko-http directly to the internet).

Also not sure if a formal discussion on the mailing list for this is excessive.

Reply:
renamed to `reset-frame-throttle...`